### PR TITLE
Fix presto-tests intermittent OOM errors

### PR DESCRIPTION
**Pull request by arhimondr**
@pnowojski @cberner @electrum

### CODE REVIEW

**Comment by arhimondr** [comment]
https://travis-ci.org/prestodb/presto/jobs/178894616

**Comment by cberner** [comment]
@pnowojski will there be sufficient test coverage of the spilling code if this is removed? Perhaps it would be better to optimize the spilling tests, instead of removing them

**Comment by pnowojski** [comment]
I'm talking with @arhimondr about this issue. It would be better to diagnose and fix this tests.

File: presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueryFramework.java [hunk 1]
```diff
@@ -44,6 +45,7 @@
 import static org.testng.Assert.assertEquals;
 import static org.testng.Assert.fail;
 
+@Listeners({TestLoggingListener.class})
```
Comment (electrum) [inline presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueryFramework.java]: The braces are not needed

File: presto-tests/src/main/java/com/facebook/presto/tests/TestLoggingListener.java [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+public class TestLoggingListener
+        extends TestListenerAdapter
+{
+    private static final Logger log = Logger.get(TestLoggingListener.class);
+
+    @Override
+    public void onTestStart(ITestResult testResult)
+    {
+        log.info("Started: %s", getTestName(testResult));
```
Comment (electrum) [inline presto-tests/src/main/java/com/facebook/presto/tests/TestLoggingListener.java]: This should say "Starting" since the test hasn't started yet.

**Comment by electrum** [comment]
Logging every single test seems pretty spammy and I'm not sure how useful it is given that they execute in parallel. I think it is enough to simply log start/finish of AbstractTestQueryFramework in the constructor and `close()` method.

**Comment by electrum** [comment]
Superceded by #6734

**Pull request closed without merging**